src/System.IO.FileSystem/tests/Directory/CreateDirectory.cs

```diff
@@ -130,7 +126,8 @@ public void ValidPathWithTrailingSlash()
             });
         }
 
-        [Fact]
+        [ConditionalFact("UsingNewNormalization")]
```

Comment:
I've tried to come up with a better name, but it comprises a lot of changes. Legacy is the term we shipped with the other way.